src/main/resources/nodeTestnet2/application.conf

```diff
@@ -0,0 +1,29 @@
+ergo {
```

Comment:
Don't think we need this file in git. Also it's a regular config file that does not provide any info related to testnet2